MINT.postman_collection.json

```diff
@@ -3034,7 +3034,7 @@
 							"response": []
 						},
 						{
-							"name": "NewMTOMilestone From Common Milestone",
+							"name": "NewMTOMilestone (Common)",
 							"event": [
 								{
 									"listen": "test",
```

Comment:
this script needs to be updated so it references the new mutation name (createMTOMilestoneCustom)
eg
"let responseData = pm.response.json().data.createMTOMilestoneCustom"


This sets a variable in the postman collection so you can call the update mutations without needing to change params

Reply:
d98f42148